docs/source/logo.rst

```diff
@@ -0,0 +1,54 @@
+TorchAudio Logo
+===============
+
+If you make your project using TorchAudio and you want to mention TorchAudio, you can use TorchAudio logo. There are couple of variations. You can download them from `here <https://download.pytorch.org/torchaudio/logo/v1.zip>`__.
+
+Please follow `the guideline <https://download.pytorch.org/torchaudio/logo/guidelines.pdf>`__ for the proper usage.
```

Comment:
this link is not currently active yet

Reply:
good catch
```suggestion
Please follow `the guideline <https://download.pytorch.org/torchaudio/logo/v1/guidelines.pdf>`__ for the proper usage.
```

Reply:
just want to check, are these guidelines applicable for general public use or is it more for internal/marketing distribution? not sure if the necessity of this or if we need to reinforce that the public uses the logo "correctly"?

Reply:
Yeah, the guideline is for any use regardless of internal/external. It's not really enforceable in practical sense, but it's to avoid the case of misuse out of good-will. Say someone wants to use it but their design criteria is a bit restricted and they cannot just place the main one. What they can do? They look at the guideline to see what they can do.